authentik/core/models.py

```diff
@@ -135,38 +160,6 @@ def is_member(self, user: "User") -> bool:
         """Recursively check if `user` is member of us, or any parent."""
         return user.all_groups().filter(group_uuid=self.group_uuid).exists()
 
-    def children_recursive(self: Self | QuerySet["Group"]) -> QuerySet["Group"]:
```

Comment:
I think for backwards compat we might want to keep this function around as a wrapper, if we can (I think there's a non-0% change it's used in some policies)